src/Span.ts

```diff
@@ -142,6 +142,12 @@ class Span implements Partial<ISpan> {
    */
 
   defaultTagsSetter(message?: TypeOfMessage): Span {
+    if (!Config.EVENT_LOGGER_TRACE_HEADERS_ENABLED) return this
```

Comment:
I think in line with the standard js/ts style, we shouldn't have single line if statements. E.g. this should be: 
```ts
if (!Config.EVENT_LOGGER_TRACE_HEADERS_ENABLED) {
  return this
}
```